src/debug_renderdoc.cpp

```diff
@@ -72,13 +40,17 @@ namespace bgfx
 			return NULL;
 		}
 
-		void* renderDocDll = bx::dlopen(
 #if BX_PLATFORM_WINDOWS
-				"renderdoc.dll"
+		// If RenderDoc is already injected in the process then use the already present DLL
+		void* renderDocDll = findModule("renderdoc.dll");
+		if (NULL == renderDocDll)
+		{
+			// Load the RenderDoc DLL from its default installation location
+			renderDocDll = bx::dlopen("C:\\Program Files\\RenderDoc\\renderdoc.dll");
```

Comment:
What if someone changed installation location?

Reply:
For example, I never install RenderDoc. I always use portable zip file to copy it in some directory that can be check-in in source control (so that other devs can use it without installation).

Reply:
Then the DLL will not be found, they will have to launch the app from RenderDoc in order to take captures (or edit the code locally to the correct location).

Reply:
I am not sure if using it without installation will work with Vulkan due to the need to register the DLL as a layer. But it's not my area of expertise. In any case I think the main thing is not to have two copies of the DLL at different paths, as then both may be loaded and that will really screw up Vulkan.

Reply:
> I am not sure if using it without installation will work with Vulkan due to the need to register the DLL as a layer.

It works, you can register from RenderDoc client. I use it this way.

Reply:
I think your fix would just work if you do:

```
	void* renderDocDll = findModule("renderdoc.dll"); <- this fixes the issue, the rest is just stays as-is.
	if (NULL == renderDocDll)
	{
		renderDocDll = bx::dlopen(
#if BX_PLATFORM_WINDOWS
				"renderdoc.dll"
#else
				"./librenderdoc.so"
#endif // BX_PLATFORM_WINDOWS
				);
	}
```

Reply:
Also it makes sense that `findModule` would return actual pointer to module, like in your original change.

Reply:
It won't find the DLL then unless it's copied next to the executable, though, right? Which Baldur says we definitely aren't supposed to do.

Reply:
I'm assuming it is important to maintain the ability to load the DLL even when not launched from RenderDoc.

Reply:
> It won't find the DLL then unless it's copied next to the executable, though, right?

Yes, but it's unrelated issue to already loaded RenderDoc you're fixing here.

That could be separate thing where we test known locations like `C:\\Program Files\\RenderDoc\\renderdoc.dll`

Reply:
OK. I can narrow down this PR to just handling the already loaded DLL and leave a comment to revisit the load path in the future.